Makefile

```diff
@@ -49,10 +53,14 @@ lint:
 	golint pkg/networkutils/*.go
 	golint ipamd/*.go
 	golint ipamd/*/*.go
+	golint controller/*.go
+	golint controller/*/*.go
 
 #go tool vet
 vet:
 	go tool vet ./pkg/awsutils
 	go tool vet ./plugins/routed-eni
 	go tool vet ./pkg/k8sapi
 	go tool vet ./pkg/networkutils
+	go tool vet controller
```

Comment:
I don't think this is working.  `go vet tool` works on files.  `go vet` works on packages.